config/default/core.extension.yml

```diff
@@ -1,24 +1,37 @@
 _core:
   default_config_hash: R4IF-ClDHXxblLcG0L7MgsLvfBIMAvi_skumNFQwkDc
 module:
+  admin_toolbar: 0
```

Comment:
These modules were enabled in the profile-base Sous version. Adding them here to keep features/functionality the same.